src/cbc_sdk/platform/policies.py

```diff
@@ -0,0 +1,1222 @@
+        self._info = json.loads(ret.text)
+        self._full_init = True
+        self._parent._on_updated_rule(self)
+
+    def _delete_object(self):
+        """
+        Deletes this rule object from the policy on the server.
+
+        Required Permissions:
+            org.policies(UPDATE)
+        """
+        if self._model_unique_id is None:
+            raise ApiError("new rule cannot be deleted")
+        self._cb.delete_object(self._parent._build_api_request_uri() + f"/rules/{self.id}")
+        self._parent._on_deleted_rule(self)
+        self._parent = None
+
+    @property
+    def is_deleted(self):
+        """Returns True if this rule object has been deleted."""
+        return self._parent is None
+
+    def validate(self):
+        """
+        Validates this rule against its constraints.
+
+        Raises:
+            InvalidObjectError: If the rule object is not valid.
+        """
+        super(PolicyRule, self).validate()
+        if not isinstance(self._info["required"], bool):
+            raise InvalidObjectError("'required' field not valid type")
+        if self._info["action"] not in PolicyRule.VALID_ACTIONS:
+            raise InvalidObjectError("'action' field value not valid")
+        if not isinstance(self._info["application"], dict) \
+           or set(self._info["application"].keys()) != PolicyRule.VALID_APP_KEYS:
+            raise InvalidObjectError("'application' field not valid type/structure")
+        if self._info["application"]["type"] not in PolicyRule.VALID_APP_TYPES:
+            raise InvalidObjectError("'application' 'type' value not valid")
+        if not isinstance(self._info["application"]["value"], str):
+            raise InvalidObjectError("'application' 'value' not valid type")
+        if self._info["application"]["type"] == "REPUTATION" \
+           and self._info["application"]["value"] not in PolicyRule.VALID_REPUTATIONS:
+            raise InvalidObjectError("'application' 'value' not valid value for type REPUTATION")
+        if self._info["operation"] not in PolicyRule.VALID_OPERATIONS:
+            raise InvalidObjectError("'operation' field value not valid")
+        return True
+
+
+"""Query Class"""
+
+
+class PolicyQuery(BaseQuery, IterableQueryMixin, AsyncQueryMixin):
+    """Query for retrieving policies (summary info only)."""
+    def __init__(self, doc_class, cb):
+        """
+        Initialize the Query object.
+
+        Args:
+            doc_class (class): The class of the model this query returns.
+            cb (CBCloudAPI): A reference to the CBCloudAPI object.
+        """
+        super(PolicyQuery, self).__init__(None)
+        self._doc_class = doc_class
+        self._cb = cb
+        self._count_valid = False
+        self._total_results = 0
+        self._policy_ids = []
+        self._system = False
+        self._system_set = False
+        self._names = []
+        self._descrs = []
```

Comment:
Can we make this `_descriptions` as `_decsrs` was not obvious as an acryonm

Reply:
Done.